pkg/comp-functions/functions/vshnredis/script/backup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# This script works both with enabled and disabled AOF. It ensures that there
+# are no AOF rewritings happening at the time of backup. If there's a dump file
+# it will also backup that one.
+
+redis_cmd() {
+  redis-cli -a "${REDIS_PASSWORD}" --cacert "${REDIS_TLS_CA_FILE}" --cert "${REDIS_TLS_CERT_FILE}" --key "${REDIS_TLS_KEY_FILE}" --tls $@
```

Comment:
```suggestion
  redis-cli -a "${REDIS_PASSWORD}" --cacert "${REDIS_TLS_CA_FILE}" --cert "${REDIS_TLS_CERT_FILE}" --key "${REDIS_TLS_KEY_FILE}" --tls "$@"
```

Input from shellcheck (And I think it's right, but I'm terrible at writing shell scripts so maybe it's not :sweat_smile: ) `Double quote array expansions to avoid re-splitting elements`

Reply:
Still seems to work as expected 👍